src/Sylius/Bundle/CoreBundle/Resources/config/state-machine.yml

```diff
@@ -180,3 +180,26 @@ winzou_state_machine:
                     to:   'shipped'
                     do:   [@sylius.email_manager.shipment, 'sendConfirmationEmail']
                     args: ['object']
+
+    sylius_review:
+        class:         %sylius.model.product_review.class%
+        property_path: status
+        graph:         sylius_product_review
+        state_machine_class: Sylius\Component\Resource\StateMachine\StateMachine
+        states:
+            new:      ~
+            accepted: ~
+            rejected: ~
+        transitions:
+            accept:
+                from: [new]
+                to:   accepted
+            reject:
+                from: [new]
+                to:   rejected
+        callbacks:
+            after:
+                update_review_subject_average_price:
+                    on: 'accept'
+                    do: [@sylius.review.updater.reviewable_average_rating, updateFromReview]
```

Comment:
I have not seen the logic of updateFromReview, but if it is recalculating everything from scratch, then we should have just ``update`` method and pass ``object.subject`` as argument. That would give use Reviewable subject.